socorro/external/postgresql/new_crash_source.py

```diff
@@ -0,0 +1,242 @@
+        )
+
+    #--------------------------------------------------------------------------
+    def close(self):
+        super(DBCrashStorageWrapperNewCrashSource, self).close()
+        self._implementation.close()
+
+    #--------------------------------------------------------------------------
+    def save_raw_crash(self, raw_crash, dumps, crash_id):
+        self._implementation.save_raw_crash(
+            raw_crash,
+            dumps,
+            crash_id
+        )
+
+    #--------------------------------------------------------------------------
+    def save_raw_crash_with_file_dumps(self, raw_crash, dumps, crash_id):
+        self._implementation.save_raw_crash_with_file_dumps(
+            raw_crash,
+            dumps,
+            crash_id
+        )
+
+    #--------------------------------------------------------------------------
+    def save_processed(self, processed_crash):
+        self._implementation.save_processed(processed_crash)
+
+    #--------------------------------------------------------------------------
+    def save_raw_and_processed(
+        self,
+        raw_crash,
+        dumps,
+        processed_crash,
+        crash_id
+    ):
+        self._implementation.save_raw_and_processed(
+            raw_crash,
+            dumps,
+            processed_crash,
+            crash_id
+        )
+
+    #--------------------------------------------------------------------------
+    def get_raw_crash(self, crash_id):
+        return self._implementation.get_raw_crash(crash_id)
+
+    #--------------------------------------------------------------------------
+    def get_raw_dump(self, crash_id, name=None):
+        return self._implementation.get_raw_dump(crash_id, name)
+
+    #--------------------------------------------------------------------------
+    def get_raw_dumps(self, crash_id):
+        return self._implementation.get_raw_dumps(crash_id)
+
+    #--------------------------------------------------------------------------
+    def get_raw_dumps_as_files(self, crash_id):
+        return self._implementation.get_raw_dumps_as_files(crash_id)
+
+    #--------------------------------------------------------------------------
+    def get_processed(self, crash_id):
+        return self._implementation.get_processed(crash_id)
+
+    #--------------------------------------------------------------------------
+    def get_unredacted_processed(self, crash_id):
+        return self._implementation.get_unredacted_processed(crash_id)
+
+    #--------------------------------------------------------------------------
+    def remove(self, crash_id):
+        self._implementation.remove(crash_id)
+
+
+#==============================================================================
+class PGPVNewCrashSource(PGQueryNewCrashSource):
+    required_config = Namespace()
+    required_config.crash_id_query = change_default(
+        PGQueryNewCrashSource,
+        'crash_id_query',
+        "select uuid from reports "
```

Comment:
@AdrianGaudebert might cry if he sees a new piece of code that's come to depend on the `reports` table.